src/Mvc/Database/DataImporter.php

```
@@ -0,0 +1,1628 @@
+	 * implementation loads the entire file into memory which works well for
+	 * typical data exports but may not scale to extremely large datasets.
+	 *
+	 * @param string $line CSV line to parse
+	 * @return array|false Parsed values or false on failure
+	 */
+	private function parseCsvLine( string $line ): array|false
+	{
+		// Use str_getcsv for parsing which handles quotes and escapes properly
+		// PHP 8.4 requires the escape parameter
+		return str_getcsv( $line, ',', '"', '' );
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'parseCsvLine'. (undefined)

(UnusedPrivateMethod)